ui/src/viewers/PdfViewerSearch.jsx

```diff
@@ -70,13 +83,20 @@ class PdfViewerSearch extends Component {
 
     return (
       <>
-        <ul>
+        <div className="PdfViewerSearch__total">
+          <ResultText result={result} />
+        </div>
+        <ul className="PdfViewerSearch__results">
           {result.results.map((res) => (
-            <li key={`page-${res.id}`}>
+            <li key={`page-${res.id}`} className="PdfViewerSearch__result">
               <p dir={dir}>
-                <Link to={this.getResultLink(res)}>
+                <a href={this.getResultLink(res)}>
                   <span
-                    className={c(Classes.ICON, `${Classes.ICON}-document`)}
+                    className={c(
+                      Classes.ICON,
+                      `${Classes.ICON}-document`,
+                      'PdfViewerSearch__icon'
+                    )}
```

Comment:
Why doesn’t this use the `Icon` component?

Reply:
For future reference:

The `document` icon from Blueprint looks like this: a single page. That’s what we want in this case.

<img width="143" alt="Screenshot 2024-10-23 at 12 53 28" src="https://github.com/user-attachments/assets/8f97ef6f-8961-4e6f-ae3e-e03f148f85cf">

However, we also have a custom set of icons, e.g. for the different FtM schemata which can be used with the `Icon` component in addition to the default Blueprint icons. As there is an FtM schema called `Document`, when using `<Icon icon="document" />` you get this custom icon which is a multi-page document:

<img width="90" alt="Screenshot 2024-10-23 at 12 53 03" src="https://github.com/user-attachments/assets/c1af404b-8be2-4fd9-ac59-e45db3934741">

Because both the custom schema icon and the Blueprint icon are named `document`, this seems to be a workaround to get the default single-page icon from Blueprint instead of the custom multi-page icon.